wp-includes/sqlite/class-wp-sqlite-translator.php

```diff
@@ -2939,6 +2939,35 @@ private function execute_alter() {
 						WP_SQLite_Token::FLAG_KEYWORD_DATA_TYPE
 					)
 				);
+
+				// Drop "FIRST" and "AFTER <another-column>", as these are not supported in SQLite.
+				$column_position = $this->rewriter->peek(
+					array(
+						'type'  => WP_SQLite_Token::TYPE_KEYWORD,
+						'value' => array( 'FIRST', 'AFTER' ),
+					)
+				);
+
+				$comma = $this->rewriter->peek(
+					array(
+						'type'  => WP_SQLite_Token::TYPE_OPERATOR,
+						'value' => ',',
+					)
+				);
+
+				if ( $column_position && ( ! $comma || $column_position->position < $comma->position ) ) {
```

Comment:
What kind of SQL syntax is this comma check intended to address? Could you provide an example?

Reply:
@brandonpayton Without the check, it can consume too much, up to the following column definitions.

E.g., for a query like this one:
```sql
ALTER TABLE _tmp_table
ADD COLUMN new1 VARCHAR(255),
ADD COLUMN new2 VARCHAR(255) FIRST
```

You would get a `SQLSTATE[HY000]: General error: 1 near ",": syntax error.`, since you've consumed up to `FIRST` from the second column when processing the first column.

Reply:
Ah, I see. I hadn't noticed that the translator loops to support the comma-separated MySQL ALTER TABLE syntax.

Let's add a test for comma-separated syntax as well.

Reply:
Thanks for explaining!

Reply:
Added a test in https://github.com/WordPress/sqlite-database-integration/pull/147/commits/2c1fe61e9c308564309bcad9ae0227d28b31a4fa.